### ISSUE TITLE
Parenthesis highlighting

### ISSUE THREAD

**Issue by fabilson-ssa** [dbeaver/dbeaver#8328]
Hi!
I have a doubt.
How can I change parenthesis highlighting?
There is a option for that?

![2020-04-03 10_23_11-DBeaver 7 0 1 - _none_ Script-16](https://user-images.githubusercontent.com/32611254/78365423-7747ea00-7595-11ea-955f-5dc5f83bb760.png)

**Comment by uslss** [comment]
hi @fabilson-ssa ,
you could change it in preferences 
![Screenshot from 2020-04-04 18-40-01](https://user-images.githubusercontent.com/31996417/78454923-d98f1080-76a3-11ea-9dac-cbbc8ed33fad.png)

**Comment by fabilson-ssa** [comment]
Uslss, at first, I would like to thank for your support and quick answer.

But, I think that maybe I didn´t make myself clear.

Let me try again.

What I am try to get is Highlight Matching {} and ()

For example: In the expression: **`select count(1) from dual`;** 

You can see a square around the first parenthesis.  I would like to know how then beginer parenthesis can be highlighted with the match one, at the end.

I will send another screenshot for example
![2020-04-06 08_48_44-Highlight Matching {} and ()](https://user-images.githubusercontent.com/32611254/78555974-7f499900-77e4-11ea-86dc-134b6970287f.png)

Thanks in advance for helping me out.

**Comment by uslss** [comment]
sorry, didn't catch you first.
but as for me, it is the element at the end is highlighted 
![image](https://user-images.githubusercontent.com/31996417/78988261-1db56180-7b39-11ea-9594-274efa29bcf4.png)

**Comment by fabilson-ssa** [comment]
Yes, that´s it I mean.
Thanks, but do you know any way of changing the rectangle around the parenthesis?

I mean, it could be in diferente backgroud color, for exemple

**Comment by serge-rider** [comment]
Generally I agree. Current way to highlight matching brackets is not good. It comes from Eclipse and sometimes it is very hard to find matching character in a long text.
Using another background instead of thin rectangle sounds like a good idea.

Let's try it. I'll use the same color as for selected words matches.

**Comment by uslss** [comment]
verified

**Comment by kkorus** [comment]
@fabilson-ssa 
Hey, have you managed to change the color of highlighted parenthesis? It doesn't look great with a dark theme currently. 

![image](https://user-images.githubusercontent.com/2054009/118358089-8e6d4f80-b57d-11eb-99af-bc96ee2cc8e2.png)

**Comment by Ryoka** [comment]
I could not find it either. It looks bad with dark theme. The color of matching brackets needs to be configurable...

**Comment by Gabriel-Wendel** [comment]
nothing to increment this feature yet?

**Comment by Xemptuous** [comment]
currently on 22.3.0 and there's still no option to change the background/foreground color of matching brackets/parentheses. Would be helpful, as with dark themes, it's almost entirely useless. I went through every color option available, and nothing. 
![image](https://user-images.githubusercontent.com/99044751/207948832-99fe56a2-c831-41cb-9c7d-f715133ffd21.png)

**Comment by philosaur** [comment]
On 23.2.0: I can't find the option to change the color or styling of parenthesis/bracket matches.

I would _swear_ this existed in a previous version...
